mcux/middleware/mcux-sdk-middleware-connectivity-framework/platform/rw61x/fwk_platform_ble.c

```diff
@@ -276,7 +276,7 @@ static void (*hci_rx_callback)(uint8_t packetType, uint8_t *data, uint16_t len);
 /*                                Public memory                               */
 /* -------------------------------------------------------------------------- */
 
-static const uint8_t hci_cal_data_params[HCI_CMD_STORE_BT_CAL_DATA_PARAM_LENGTH] = {
```

Comment:
Curious but why is this a problem?

Reply:
This is done to be able to use those buffers inside the hci_nxp driver on Zephyr.

Reply:
Okay, I see the Zephyr PR referencing these directly so I understand. 

We should decide generally a common method for accessing these or the underlying function being done with these so we don't have to patch this each time we update the Zephyr NXP HAL.

Reply:
@dleach02 yes this is something we want to address, we'll check how we can backport this patch to the framework.